frontend/src/components/MeetingRoom/ReportIssue/FeedbackForm/FilePicker/FilePicker.tsx

```diff
@@ -63,7 +63,7 @@ const FilePicker = ({
       const screenshotData = await captureScreenshot();
       setImageSrc(screenshotData);
       onFileSelect(screenshotData);
-    } catch (error) {
+    } catch (_error) {
```

Comment:
I think we're seeing some SC warnings. Can we add a `// NOSONAR` here to see if the issue goes away?

Reply:
for this one linter won't let me add `// NOSONAR` on the same line as `catch(_error).` I don't think this is worth logging to the console, though. Even though Sonar might complain, it's still passing the CI looks like so I think I might leave this particular one as-is
on the other ones I added some console logging